system-variables.md

```diff
@@ -1094,6 +1088,14 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 
 ### tidb_ddl_enable_fast_reorg <span class="version-mark">New in v6.3.0</span>
 
+<CustomContent platform="tidb-cloud">
+
+> **Note:**
+>
+> To improve the speed for index creation using this variable, make sure that your TiDB cluster is hosted on AWS and your TiDB node size is at least 8 vCPU. For [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta), this feature is unavailable.
```

Comment:
Add this limitation in https://docs.pingcap.com/tidbcloud/serverless-tier-limitations

Reply:
added in https://github.com/pingcap/docs/pull/12041/commits/e085bbd4198e6faea28b7ff5abc9b7d799ed2d81